Frontend/ui-customer/index.js

```diff
@@ -208,6 +211,7 @@ window.onResumeListeners = [];
 
 window.onPause = function () {
   console.error("onEvent onPause");
+  // JBridge.pauseAudioPlayer();
```

Comment:
how is it handled now?

Reply:
This is separate audio player instance. as it is SOS audio we can't turn off in onPause. we have a button to turn off or if app is killed it will be stopped.

Reply:
it was not used for any other use-case anywhere else before?